docs/serverless-v2/06-01-function-cr.md

```diff
@@ -0,0 +1,77 @@
+
+```bash
+kubectl get crd functions.serverless.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+The following Function object creates a lambda which runs on the Node.js 8 runtime and responds to HTTP requests with "Hello World".
+
+```yaml
+apiVersion: serverless.kyma-project.io/v1alpha1
+kind: Function
+metadata:
+  name: my-test-lambda
+  namespace: default
+spec:
+  functionContentType: plaintext
+  runtime: nodejs8
+  timeout: 360
+  env:
+  - name: PERSON_NAME
+    value: "John"
+  deps: |
+    {
+      "name": "hellowithdeps",
+      "version": "0.0.1",
+      "dependencies": {
+        "end-of-stream": "^1.4.1",
+        "from2": "^2.3.0",
+        "lodash": "^4.17.5"
+      }
+    }
+  function: |
+    module.exports = {
+      main: function(event, context) {
+        const name = process.env.PERSON_NAME;
+        return 'Hello ' + name;
+      }
+    }
+```
+
+## Custom resource properties
+
+This table lists all the possible properties of a given resource together with their descriptions:
+
+| Property | Required | Description |
+|----------|:---------:|-------------|
+| **metadata.name** | Yes | Specifies the name of the CR. |
+| **metadata.namespace** | Yes | Defines the Namespace in which the CR is available. |
+| **spec.functionContentType** | Yes | Specifies the content type of the lambda's code defined in the **function** property. The content type can be either `plaintext` or `base64`. |
+| **spec.runtime** | Yes | Specifies the software runtime used to run the lambda's code. It can be either `nodejs6` or `nodejs8`. |
+| **spec.timeout** | No | Specifies the duration in seconds after which the lambda execution is terminated. The default value is `180`. |
+| **spec.env** | No | Specifies environment variables you need to export for the lambda. |
+| **spec.deps** | No | Specifies the lambda's dependencies. |
+| **spec.function** | Yes | Provides the lambda's source code. |
+
+## Related resources and components
+
+The Function custom resource relies on resources from [Knative Serving](https://knative.dev/docs/serving/) and [Tekton Pipelines](https://github.com/tektoncd/pipeline):
+
+| Resource | Description |
+|----------|-------------|
+|[KService CR](https://github.com/knative/docs/blob/master/docs/serving/spec/knative-api-specification-1.0.md#service) | Orchestrates the deployment and availability of the function.|
+|[TestRun CR](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md) | Builds a container image containing the function code together with its configured runtime. |
+
+These components use this CR:
+
+| Component   |   Description |
+|-------------|---------------|
+| Serverless |  Uses the Function CR for the detailed lambda definition, including the environment on which it should run. |
```

Comment:
![image](https://user-images.githubusercontent.com/20404945/74652126-a50fd580-5185-11ea-9713-92308836a60d.png)

We don't have `Serverless` component. Can you describe what do you mean?

Reply:
Maybe `Function Controller`?